src/compiler/parser.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path="utilities.ts"/>
 
 module ts {
+    export var throwOnJSDocErrors = false;

```

Comment:
This has been removed.  It was only for ascertaining what we could'nt handle on code in the wild.